lifecycle-operator/chart/templates/lifecycle-mutating-webhook-configuration.yaml

```diff
@@ -34,6 +35,11 @@ webhooks:
       - keptn-lifecycle-toolkit-system
       - observability
       - monitoring
+{{- else }}
```

Comment:
I would add an extra entry called "kube-api" because we know we should not mutate manifests in these ns: 
- kube-system
- kube-public
- kube-node-lease